tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java

```diff
@@ -491,14 +491,14 @@ private static String valueToString(Object value)
 
     public static class AcceptableValues
     {
-        private final List<Object> values;
+        private final List<?> values;
 
-        public AcceptableValues(List<Object> values)
+        public AcceptableValues(List<?> values)
         {
             this.values = unmodifiableList(new ArrayList<>(requireNonNull(values, "values can not be null")));
         }
 
-        public List<Object> getValues()
+        public List<?> getValues()
```

Comment:
How is this used by callers?

Reply:
In Presto codebase there is one place where the returned value is assigned to `List<Object>`. This will need to be changed.